src/platform/log/zephyr/log.c

```diff
@@ -40,3 +82,76 @@ mender_log_exit(void) {
     /* Nothing to do */
     return MENDER_OK;
 }
+
+#ifdef CONFIG_MENDER_DEPLOYMENT_LOGS
+mender_err_t
+mender_deployment_logs_activate(void) {
+    log_backend_activate(&log_backend_mender_depl_logs, NULL);
+    return MENDER_OK;
+}
+
+mender_err_t
+mender_deployment_logs_deactivate(void) {
+    MENDER_NDEBUG_UNUSED mender_err_t ret;
+
+    /* In case there is an incomplete log message, save it. */
+    if (0 != msg_tmp_idx) {
+        msg_tmp[msg_tmp_idx++] = '\0';
```

Comment:
Should we append `...` or something similar to signal that it is incomplete? Maybe not worth the extra code 🤔

Reply:
Yeah, I think it will be pretty obvious. We can revisit this if there's any need.